Library/Formula/globus-toolkit.rb

```diff
@@ -12,14 +10,22 @@ class GlobusToolkit < Formula
     sha1 "1cbbe7a3d08711cbc5665e91558417d7b2028d3a" => :mountain_lion
   end
 
+  depends_on "openssl"
   depends_on "libtool" => :run
+  depends_on "pkg-config" => :build
+
+  option "with-check", "Test the toolkit when installing"
 
   def install
     ENV.deparallelize
     ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
-    system "./configure", "--prefix=#{prefix}"
+    man.mkpath
+    system "./configure", "--prefix=#{prefix}",
+                          "--mandir=#{man}",
+                          "--disable-dependency-tracking"
     system "make"
-    system "make install"
-    share.install prefix/'man'
+    system "make", "check" if build.with? "check"
+    system "make", "install"
+    share.install prefix/"man"
```

Comment:
I don’t know. How would we resolve such a conflict? Patch all formulae (there’re only two of them)?

I just edited the formulae to use `bin.write_exec_script` as you suggested instead of `rm_rf`ing `bin/ssh.sd`.